### ISSUE TITLE
Final peer review - jk2332

### ISSUE THREAD

**Issue by jk2332** [KyleWadell/YoutubeTrendingProject#11]
This project's goal is to classify the category of a Youtube video. They used dataset from Kaggle. 

What I like:
The report structure is concise and coherent. 
They use sufficient visualizations to help understanding. 
They have good explanation to how they tried out sufficient number of different models to give the best accuracy

For improvement:
Information on how the data is being preprocessed and how that affects the performance could be incorporatted
I think in general, the model is under-fitted to the dataset. There needs to be more info on how to improve this (maybe with trying out more complex model, changing hyper-parameters, etc) 
I think more loss functions and different regularizers could be tried out and explained in terms of performance
